### PR TITLE
docs: add Chinese translation for act.mdx

### LINKED ISSUE #3290: 🌐 [i18n-zh] Translating docs to Chinese
### Ticket Type

🧹 Maintenance / Documentation

## Description

This issue tracks the progress of translating the LeRobot documentation from English to Chinese (zh-Hant for now — collaborations are welcome, e.g. zh-Hans).

The goal is to make LeRobot more accessible to Chinese-speaking users and contributors by providing localized documentation. This issue serves as a central reference point for tracking which pages are being translated, their corresponding PRs, and which pages are not yet being worked on.

## Translation Progress

> Directory: `docs/source/zh/`

### Get Started
- [ ] `index.mdx` — LeRobot
- [ ] `installation.mdx` — Installation

### Tutorials
- [ ] `il_robots.mdx` — Imitation Learning for Robots
- [ ] `bring_your_own_policies.mdx` — Bring Your Own Policies
- [ ] `integrate_hardware.mdx` — Bring Your Own Hardware
- [ ] `hilserl.mdx` — Train a Robot with RL
- [ ] `hilserl_sim.mdx` — Train RL in Simulation
- [ ] `multi_gpu_training.mdx` — Multi GPU Training
- [ ] `hil_data_collection.mdx` —  Human-In-the-Loop Data Collection
- [ ] `peft_training.mdx` — Training with PEFT (e.g., LoRA)
- [ ] `rename_map.mdx` — Using Rename Map and Empty Cameras

### Datasets
- [ ] `lerobot-dataset-v3.mdx` — Using LeRobotDataset
- [ ] `porting_datasets_v3.mdx` — Porting Large Datasets
- [ ] `using_dataset_tools.mdx` — Using the Dataset Tools
- [ ] `dataset_subtask.mdx` — Using Subtasks in the Dataset
- [ ] `streaming_video_encoding.mdx` — Streaming Video Encoding

### Policies
- [ ] `act.mdx` — ACT
- [ ] `smolvla.mdx` — SmolVLA
- [ ] `pi0.mdx` — π₀ (Pi0)
- [ ] `pi0fast.mdx` — π₀-FAST
- [ ] `pi05.mdx` — π₀.₅
- [ ] `groot.mdx` — NVIDIA GR00T N1.5
- [ ] `xvla.mdx` — X-VLA
- [ ] `multi_task_dit.mdx` — Multitask DiT Policy
- [ ] `walloss.mdx` — WALL-OSS

### Reward Models
- [ ] `sarm.mdx` — SARM

### Inference
- [ ] `async.mdx` — Use Async Inference
- [ ] `rtc.mdx` — Real-Time Chunking (RTC)

### Simulation
- [ ] `envhub.mdx` — Environments from the Hub
- [ ] `envhub_leisaac.mdx` — Control & Train Robots in Sim (LeIsaac)

### Benchmarks
- [ ] `adding_benchmarks.mdx`  — Adding a New Benchmark
- [ ] `libero.mdx` — Using Libero
- [ ] `metaworld.mdx` — Using MetaWorld
- [ ] `envhub_isaaclab_arena.mdx` — NVIDIA IsaacLab Arena Environments

### Robot Processors
- [ ] `introduction_processors.mdx` — Introduction to Robot Processors
- [ ] `debug_processor_pipeline.mdx` — Debug Your Processor Pipeline
- [ ] `implement_your_own_processor.mdx` — Implement Your Own Processor
- [ ] `processors_robots_teleop.mdx` — Processors for Robots and Teleoperators
- [ ] `env_processor.mdx` — Environment Processors
- [ ] `action_representations.mdx` — Action Representations

### Robots
- [ ] `so101.mdx` — SO-101
- [ ] `so100.mdx` — SO-100
- [ ] `koch.mdx` — Koch v1.1
- [ ] `lekiwi.mdx` — LeKiwi
- [ ] `hope_jr.mdx` — Hope Jr
- [ ] `reachy2.mdx` — Reachy 2
- [ ] `unitree_g1.mdx` — Unitree G1
- [ ] `earthrover_mini_plus.mdx` — Earth Rover Mini
- [ ] `omx.mdx` — OMX
- [ ] `openarm.mdx` — OpenArm

### Teleoperators
- [ ] `phone_teleop.mdx` — Phone

### Sensors
- [ ] `cameras.mdx` — Cameras

### Supported Hardware
- [ ] `torch_accelerators.mdx` — PyTorch Accelerators

### Resources
- [ ] `notebooks.mdx` — Notebooks
- [ ] `feetech.mdx` — Updating Feetech Firmware
- [ ] `damiao.mdx` — Damiao Motors and CAN Bus

### About
- [ ] `backwardcomp.mdx` — Backward Compatibility
- [ ] `contributing.md` — How to Contribute to LeRobot

## Checklist

- [x] I have searched existing tickets to ensure this isn't a duplicate.
- [x] I am using the latest version of the `main` branch.
- [x] I have verified this is not an environment-specific problem.

## Additional Info / Workarounds

Related issues: https://github.com/huggingface/lerobot/issues/3058

### PR DESCRIPTION
**Pull request by yuzegong746-cmd**
## Description
Add Simplified Chinese (zh-Hans) translation of multiple documentation files.

This PR is part of the i18n effort to provide Chinese documentation for LeRobot users.

The translation covers:
- `act.mdx` — ACT (Policies)
- `using_dataset_tools.mdx` — Using Dataset Tools (Datasets)
- `async.mdx` — Async Inference (Inference)
- `rtc.mdx` — Real-Time Chunking (Inference)
- `lerobot-dataset-v3.mdx` — LeRobotDataset v3.0 (Datasets)
- `porting_datasets_v3.mdx` — Porting Large Datasets (Datasets)
- `dataset_subtask.mdx` — Using Subtasks (Datasets)
- `streaming_video_encoding.mdx` — Streaming Video Encoding (Datasets)

Closes #3290

## Notes
- Followed the structure and format of #3293 (tc-huang's Chinese translation PR)
- Kept all code blocks, commands, and parameter names in English
- Kept all hyperlinks intact

### CODE REVIEW

File: docs/source/zh-hans/act.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+2. **训练时长**：单块 GPU 训练 10 万步大约需要数小时
+3. **批量大小**：建议从 8 开始，根据显存情况适当调整
+
+### 使用 Google Colab 训练
+
+如果本地没有性能足够的 GPU，可以使用 Google Colab 进行云端训练，具体步骤参考 [ACT 训练 Notebook](./notebooks#training-act)。
+
+## 评估 ACT
+
+训练完成后，使用 `lerobot-record` 命令配合已训练的策略进行评估。该命令会运行推理并记录评估片段：
```
Comment (leledeyuan00) [inline docs/source/zh-hans/act.mdx]: nit: “对策略评估”比"配合策略评估"稍微语句通顺一些。
```suggestion
训练完成后，可使用 `lerobot-record` 命令对已训练的策略进行评估。该命令会运行推理并记录评估片段：
```

File: docs/source/zh-hans/act.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+1. **视觉骨干网络**：使用 ResNet-18 处理来自多个相机视角的图像
+2. **Transformer 编码器**：融合相机特征、关节位置以及学习到的潜变量
+3. **Transformer 解码器**：通过交叉注意力机制生成连贯的动作序列
+
+策略的输入包括：
+
+- 多路 RGB 图像（如腕部相机、前置或顶置相机）
+- 机器人当前各关节的位置
+- 潜在风格变量 `z`（训练时学习，推理时置零）
```
Comment (leledeyuan00) [inline docs/source/zh-hans/act.mdx:43]: nit: ”训练时学习“读起来稍有点生硬，要不要修改一下？
```suggestion
潜在风格变量 `z`（在训练阶段学习，在推理阶段置零）
```

File: docs/source/zh-hans/async.mdx [hunk 3]
```diff
@@ -0,0 +1,308 @@
+# 异步推理
+
+在我们的 [SmolVLA](https://huggingface.co/papers/2506.01844) 中，我们引入了一种在真实机器人上运行推理的新方式——**将动作预测与动作执行解耦**。
+本教程将介绍如何使用异步推理（_async inference_），支持 SmolVLA 的微调版本以及 LeRobot 所支持的所有策略。
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx:4]: nit: 这里说“通过SmolVLA来介绍异步推理”会不会更通顺一些？
```suggestion
本教程将通过SmolVLA的微调版本来介绍如何使用异步推理（_async inference_），其同样适用于 LeRobot 所支持的所有策略。
```

File: docs/source/zh-hans/async.mdx [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+**所有 LeRobot 支持的策略均可使用异步推理！**
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx:5]: nit: 这里我觉得是强调让人去动手尝试。（个人翻译倾向）
```suggestion
**尝试将异步推理应用在所有LeRobot 支持的策略中吧！**
```

File: docs/source/zh-hans/async.mdx [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+
+**你将学到：**
+
+1. 为什么异步推理很重要，以及它与传统顺序推理的区别。
+2. 如何在同一台机器上，甚至跨网络启动 `PolicyServer` 并连接 `RobotClient`。
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx:10]: sug: “跨网络”这里我觉得有些歧义了，这里应该还是处于一个网络中，只不过是通过该网络多设备进行通信的意思。
```suggestion
2. 如何在同一台设备中，甚至网络中多台设备上启动 `PolicyServer` 并连接 `RobotClient`。
```

File: docs/source/zh-hans/async.mdx [hunk 6]
```diff
@@ -0,0 +1,308 @@
...
+3. 如何为你的机器人和策略调整关键参数（`actions_per_chunk`、`chunk_size_threshold`）。
+
+如果遇到问题，欢迎加入我们的 [Discord 社区](https://discord.gg/s3KuuzsPFb)！
+
+简而言之：使用_异步推理_时，机器人在持续执行动作的同时，策略服务器已在计算下一批动作——从而消除"等待推理"的延迟，实现更流畅、更快速响应的行为。
+这与同步推理（sync）有本质区别——在同步推理中，机器人在策略计算下一批动作时会保持空闲。
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx:16]: nit: “空闲”->”等待“
```suggestion
这与同步推理（sync）有本质区别——在同步推理中，机器人在策略计算下一批动作时会保持等待。
```

File: docs/source/zh-hans/async.mdx [hunk 7]
```diff
@@ -0,0 +1,308 @@
...
+
+你可以在我们的[博客文章](https://huggingface.co/blog/async-robot-inference)中了解更多关于异步推理的信息。本指南旨在帮助你快速在自己的环境中配置并运行异步推理。
+
+首先，使用 `async` 标签安装 `lerobot`，以安装运行异步推理所需的额外依赖。
+
+```shell
+pip install -e ".[async]"
+```
+
+然后，启动一个策略服务器（在一个终端中，或在另一台机器上），并指定客户端连接所需的主机地址和端口。
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx]: nit: 个人觉得”机器“换成”设备“更贴切一些。
```suggestion
然后，启动一个策略服务器（在一个终端中，或在另一台设备上），并指定客户端连接所需的主机地址和端口。
```

File: docs/source/zh-hans/async.mdx [hunk 8]
```diff
@@ -0,0 +1,308 @@
...
+运行以下命令启动策略服务器：
+
+```shell
+python -m lerobot.async_inference.policy_server \
+     --host=127.0.0.1 \
+     --port=8080
+```
+
+这将在 `127.0.0.1:8080`（即 `localhost`，端口 8080）上启动一个策略服务器。此时策略服务器为空，所有关于运行哪个策略及其参数的信息将在客户端与服务器的首次握手中指定。使用以下命令启动客户端：
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx:39]: nit: "所有关于运行哪个"，个人觉得读起来有点不太通顺。
```suggestion
这将在 `127.0.0.1:8080`（即 `localhost`，端口 8080）上启动一个策略服务器。此时策略服务器为空，其相关策略及其参数的所有信息将在客户端与服务器的首次握手中指定。使用以下命令启动客户端：
```

File: docs/source/zh-hans/async.mdx [hunk 9]
```diff
@@ -0,0 +1,308 @@
...
+    --policy_type=your_policy_type \ # POLICY：要运行的策略类型（smolvla、act 等）
+    --pretrained_name_or_path=user/model \ # POLICY：服务器上模型的名称/路径（如 lerobot/smolvla_base）
+    --policy_device=mps \ # POLICY：服务器上运行策略所使用的设备（cuda、mps、xpu、cpu）
+    --actions_per_chunk=50 \ # POLICY：每次输出的动作数量
+    --chunk_size_threshold=0.5 \ # CLIENT：向服务器发送新观测前的队列大小阈值
+    --aggregate_fn_name=weighted_average \ # CLIENT：用于聚合重叠部分动作的函数
+    --debug_visualize_queue_size=True # CLIENT：是否在运行时可视化队列大小
+```
+
+总结来说，你需要为以下各项指定参数：
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx:58]:
```suggestion
总体来说，你需要为以下各项指定参数：
```

File: docs/source/zh-hans/async.mdx [hunk 10]
```diff
@@ -0,0 +1,308 @@
...
+
+- `SERVER`：策略服务器的地址和端口
+- `ROBOT`：要连接的机器人类型、连接端口以及机器人的本地 `id`
+- `POLICY`：要运行的策略类型，以及服务器上模型检查点的名称/路径。还需指定服务器使用的设备，以及每次输出的动作数量（不超过策略支持的最大动作数）
+- `CLIENT`：向服务器发送新观测前的队列大小阈值，以及用于聚合重叠部分动作的函数。可选地，也可在运行时可视化队列大小，以便调整 `CLIENT` 参数
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx:63]: sug: ”可选地“有点不太自然。
```suggestion
- `CLIENT`：向服务器发送新观测前的队列大小阈值，以及用于聚合重叠部分动作的函数。此外，也可自行选择在运行时可视化队列大小，以便调整 `CLIENT` 参数
```

File: docs/source/zh-hans/async.mdx [hunk 11]
```diff
@@ -0,0 +1,308 @@
...
+随着机器人模型规模的增大，这一问题有加剧的风险。
+
+<p align="center">
+  <img
+    src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/async-inference/sync.png"
+    width="80%"
+  ></img>
+</p>
+<p align="center">
+  <i>同步推理</i>使机器人在策略计算下一批动作时保持空闲。
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx:86]: nit: 使用”等待“代替”空闲“更容易理解。
```suggestion
  <i>同步推理</i>使机器人在策略计算下一批动作时保持等待。
```

File: docs/source/zh-hans/async.mdx [hunk 12]
```diff
@@ -0,0 +1,308 @@
...
+</p>
+
+为解决这一问题，我们设计了异步推理范式——将动作规划与执行解耦，从而实现（1）更高的适应性，以及最重要的（2）无空闲帧。
+关键在于，使用异步推理时，下一批动作在当前批次耗尽_之前_就已计算完毕，从而消除空闲。
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx:90]: nit: 关于”空闲“和”等待“
```suggestion
关键在于，使用异步推理时，下一批动作在当前批次耗尽_之前_就已计算完毕，从而消除等待。
```

File: docs/source/zh-hans/async.mdx [hunk 13]
```diff
@@ -0,0 +1,308 @@
...
+通过对重叠部分的不同动作块进行聚合，可获得最新的规划和更紧密的控制循环，从而保证更高的适应性。
+
+<p align="center">
+  <img
+    src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/async-inference/async.png"
+    width="80%"
+  ></img>
+</p>
+<p align="center">
+  <i>异步推理</i>因在当前批次耗尽前就计算好下一批次，从而实现无空闲。
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx:100]: nit: 关于”空闲“和”等待“
```suggestion
  <i>异步推理</i>因在当前批次耗尽前就计算好下一批次，从而无需等待。
```

File: docs/source/zh-hans/async.mdx [hunk 14]
```diff
@@ -0,0 +1,308 @@
...
+</p>
+
+---
+
+## 启动策略服务器
+
+策略服务器是对 `PreTrainedPolicy` 的封装，负责与来自机器人客户端的观测进行交互。
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx]: sug: ”与观测进行交互“读起来有点不太自然。
```suggestion
策略服务器是对 `PreTrainedPolicy` 的封装，使其能够接收来自机器人客户端的观测数据。
```

File: docs/source/zh-hans/async.mdx [hunk 15]
```diff
@@ -0,0 +1,308 @@
...
+
+<Tip>
+  `actions_per_chunk` 和 `chunk_size_threshold` 的不同取值会产生不同的行为。
+</Tip>
+
+增大 `actions_per_chunk` 可以降低动作队列为空的概率，因为计算新批次时队列中有更多动作可用。
+但过大的 `actions_per_chunk` 也可能导致动作精度下降，因为在更长时间跨度上预测动作会带来误差累积。
+
+增大 `chunk_size_threshold` 则会更频繁地向 `PolicyServer` 发送观测以触发推理，从而产生更多动作块更新，这些更新在较大的重叠部分上聚合，带来更高的适应性。极端情况下，每个观测都会触发一次动作块预测，而当前批次只消耗极少动作，新批次随即生成。
+此选项也会对推理管线造成更大压力。相反，`chunk_size_threshold` 接近 0.0 时则退化为同步推理——只有当前批次耗尽时才发送新观测。
```
Comment (leledeyuan00) [inline docs/source/zh-hans/async.mdx]: sug: 这一段原文很长，逻辑有些复杂，建议再重新整理下。下文是我的建议：
```suggestion
一方面，增大 `actions_per_chunk`可以使得在计算得到新的动作批次时有更多的动作可被使用，从而降低无动作可用的概率。
但过大的 `actions_per_chunk` 也可能会在更长时间跨度的动作预测上累积误差，最终造成动作精度的下降。

另一方面，增大 `chunk_size_threshold` 则会更频繁地向 `PolicyServer` 发送观测数据以进行推理，从而产生更多动作块更新，以生成更大的重叠部分。这使得其在极端情况下得到了高适应性，即每个观测数据都会触发一次动作块预测，并且当该动作块中的动作刚被消耗一点，立马就会有新的动作块生成出来。
由于此选项频繁多次进行请求，会给推理管线（pipeline）带来更多的压力。反之，`chunk_size_threshold` 接近 0.0 时则退化为同步推理——只有当前批次耗尽时才发送新观测。
```
